src/router/index.js

```diff
@@ -44,11 +44,10 @@ export default route(async (/* { store, ssrContext } */) => {
   }
 
   Router.beforeEach(async (to, from, next) => {
-    const authRoutes = ['SilentRefresh', 'Redirect', 'Login'];
     const isComingFromGithub = window.location.href.search(/\?code=(.*)/) !== -1;
     const isUserReady = getUserSessionToken() || isComingFromGithub || !process.env.IFBACKEND;
 
-    if (!authRoutes.includes(to.name) && !isUserReady && process.env.IFBACKEND) {
+    if (!isUserReady && process.env.IFBACKEND) {
```

Comment:
put all of this file modification is the commit you update the router.

Reply:
done